### PR TITLE
[WIP] add PyWPS

### PR DESCRIPTION
**Pull request by tomkralidis**
This PR adds PyWPS 3.2.5 to OSGeo-Live via debian package and creates ancillary files via install script along with a processes bootstrap for proof of concept.

**Note**: I'll squash the PR once it's reviewed and fixes are made.

### CODE REVIEW

**Comment by kalxas** [comment]
@tomkralidis the Xenial code has just landed in master.
We will need to re-base this branch.

**Comment by tomkralidis** [comment]
@kalxas done.

**Comment by kalxas** [comment]
thanks, will enable it on the next nightly build

File: bin/install_pywps.sh [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+Comment=PyWPS
+Categories=Application;Education;Geography;WPS
+Exec=firefox $PYWPS_URL?service=WPS&version=1.0.0&request=GetCapabilities
+Icon=/usr/local/share/icons/$ICON_NAME
+Terminal=false
+StartupNotify=false
+Categories=Education;Geography;
+EOF
+
+cp $PYWPS_DESKTOP "$USER_HOME/Desktop/Web Services"
```
Comment (kalxas) [inline bin/install_pywps.sh]: The Desktop folder "Web Services" does not exist at this point in the build.
The desktop launcher should just land in "$USER_HOME/Desktop/"

File: bin/install_pywps.sh [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+chown "$USER_NAME.$USER_NAME" "$USER_HOME/Desktop/Web Services/pywps.desktop"
```
Comment (kalxas) [inline bin/install_pywps.sh]: Please also update this according to the above line

**Comment by kalxas** [comment]
The build result of this PR is located here:
http://aiolos.survey.ntua.gr/gisvm/dev/osgeo-live-nightly-PR112-amd64-dbe8196.iso
And the build logs:
http://aiolos.survey.ntua.gr/gisvm/dev/osgeo-live-nightly-PR112-amd64-dbe8196-log.tar.gz

The desktop link returns 404

**Comment by tomkralidis** [comment]
@kalxas issues addressed and rebased. I'm not sure why the desktop link would be 404.  Having said this, given the PyWPS desktop points to a WSGI wrapper. the Apache install needs `libapache2-mod-wsgi` (not sure if `a2enmod wsgi` is also required), and the Apache instance likely needs rebooting.  I don't know enough about the OSGeoLive setup to assess whether this is required in the PyWPS install script per se of elsewhere given this affects other parts.

**Comment by tomkralidis** [comment]
@kalxas poking around other install scripts, it appears that this is done as required.  I've added commands to enable mod_wsgi, the PyWPS apache config, restarting Apache.

**Comment by kalxas** [comment]
Thanks @tomkralidis 
I will check the next build and report back

**Pull request merged**